art/step/checks.py

```diff
@@ -1,40 +1,104 @@
 import math
 from abc import ABC, abstractmethod
 from dataclasses import dataclass, field
-from typing import List
 
 
 @dataclass
 class ResultOfCheck:
+    """
+    Dataclass representing the result of a check operation.
+
+    Attributes:
+        is_positive (bool): Indicates if the check was successful. Defaults to True.
+        error (str): Error message if the check was not successful. Defaults to an empty string.
+    """
     is_positive: bool = field(default=True)
     error: str = field(default="")
 
 
 class Check(ABC):
-    name: str
-    description: str
-    required_files: List[str]
```

Comment:
Are the attributes removed on purpose? There is no __innit_ function.
They are described in the commend bellow them

Reply:
Good catch, probably a merge error. There's no init as it's an abstract class (ABC)